Meadowlark-js/packages/meadowlark-core/src/middleware/LogAnonymizedRequestMiddleware.ts

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { isDebugEnabled, Logger } from '@edfi/meadowlark-utilities';
+import { writeRequestToLog } from '../Logger';
+import { MiddlewareModel } from './MiddlewareModel';
+
+const moduleName = 'core.middleware.ParseBodyMiddleware';
+
+export async function anonymizeAndLogRequestBody({
+  frontendRequest,
+  frontendResponse,
+}: MiddlewareModel): Promise<MiddlewareModel> {
+  // if there is a response already posted, we are done
+  if (frontendResponse != null) return { frontendRequest, frontendResponse };
+
+  if (isDebugEnabled() && !frontendRequest.middleware.resourceInfo.isDescriptor) {
+    writeRequestToLog(moduleName, frontendRequest, 'anonymizeAndLogRequestBody');
```

Comment:
In this case, we're about to write a debug message. Don't think we need to also write an info message indicating that we entered the block of code.

Reply:
Just following standard practice but had the same thought, and agree it's not needed, will remove